agent/php_internal_instrument.c

```diff
@@ -796,6 +800,34 @@ static void nr_php_instrument_datastore_operation_call(
   }
 }
 
+/* 
+ * Like nr_php_instrument_datastore_operation_call() but does not
+ * call the underlying internal function
+ */
+static void nr_php_instrument_datastore_operation(
+    nr_datastore_t datastore,
+    const char* operation,
+    nr_datastore_instance_t* instance,
+    bool instance_only,
+    INTERNAL_FUNCTION_PARAMETERS) {
```

Comment:
Is this argument to the function necessary? It doesn't seem like we're using any of the `INTERNAL_FUNCTION_PARAMETERS` (`execute_data`, `return_value`) here.

Reply:
removed in [c8c3637](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/c8c3637f08b6375f7c92902dada51b124ba977e6)